.github/workflows/release.yml

```
@@ -0,0 +1,73 @@
+name: Release with SLSA provenance
+
+on:
+  push:
+    tags:
+      - 'v*'
+    branches:
+      - 'release/**'
+
+permissions:
+  contents: read
+  packages: write
+  id-token: write # Required for SLSA provenance generation
```

from SLSA_Security_Expert
GitHub Tokenのパーミッションスコープをより制限的にすべきです。現在の設定では必要以上の権限が付与されている可能性があります。

```suggestion
permissions:
  contents: write # Required for creating releases
  packages: write # Required for pushing to GHCR
  id-token: write # Required for SLSA provenance
  security-events: write # Required for Trivy scanning
```